README.md

```diff
@@ -1,21 +1,52 @@
+| 3   | [model_selection_demo.ipynb](model_selection_demo.ipynb)                                       | Perform hyperparameter optimization to find the best settings of cleanlab's optional parameters.                                                                                                                                                                             |
+| 4   | [simplifying_confident_learning_tutorial.ipynb](simplifying_confident_learning_tutorial.ipynb) | Implement cleanlab as raw numpy code.                                                                                                                                                                                                                                        |
+| 5   | [visualizing_confident_learning.ipynb](visualizing_confident_learning.ipynb)                   | Demonstrate how cleanlab performs noise matrix estimation.                                                                                                                                                                                                                   |
+| 6   | [cifar10-cnn-coteaching](cifar10-cnn-coteaching)                                               | Demonstrate the use of two experimental modules from cleanlab: [cifar_cnn.py](https://github.com/cleanlab/cleanlab/blob/master/cleanlab/experimental/cifar_cnn.py) and [coteaching.py](https://github.com/cleanlab/cleanlab/blob/master/cleanlab/experimental/coteaching.py) |
+| 7   | [mnist-cnn](mnist-cnn)                                                                         | Demonstrate the use of the following experimental module from cleanlab: [mnist_pytorch.py](https://github.com/cleanlab/cleanlab/blob/master/cleanlab/experimental/mnist_pytorch.py)                                                                                          |
+| 8   | [amazon-reviews-fasttext](amazon-reviews-fasttext)                                             | Demonstrate the use of the following experimental module from cleanlab: [fasttext.py](https://github.com/cleanlab/cleanlab/blob/master/cleanlab/experimental/fasttext.py)                                                                                                    |
+
+## Instructions
+
+To run the latest example notebooks, execute the commands below which will install the required libraries in a virtual environment.
+
+To run the examples with your own version of cleanlab, simply edit the first line of requirements.txt, or delete it and install cleanlab separately.
+
+```console
+$ python -m pip install virtualenv
+$ python -m venv env
+$ source env/bin/activate
+$ python -m pip install -r requirements.txt
+```
+
+For examples 1-5, you may run the notebooks individually or run the bash script below which will execute and save each notebook.
+
+Bash script:
+
+```console
+$ ./run_all_notebooks.sh
```

Comment:
may be better as:
```
bash ./run_all_notebooks.sh
```

Some systems throw permissions issues when the `bash` is omitted.

Reply:
Sounds good, agree. I made this change.